server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+require('dotenv').config();
+const app = new express();
+const port = process.env.PORT || 5000;
+
+app.use(express.static('public'));
+
+app.get('/', (req, res) => {
+  res.sendFile(__dirname + '/');
```

Comment:
Please be sure to set the proper cache control headers, right now this is sending `Cache-Control: public, max-age=0` which is not cacheable. Maybe add:

```js
res.setHeader('Cache-Control', 'public, max-age=300')
```

For a 5min timeout similar to the live site currently in production.

Reply:
Resolved via 98116e5